mpqp/tools/circuit.py

```diff
@@ -27,16 +25,17 @@
 def random_circuit(
     gate_classes: Optional[list[type]] = None,
     nb_qubits: int = 5,
-    nb_gates: int = np.random.randint(5, 10),
+    nb_gates: Optional[int] = None,
+    seed: Optional[Union[int, np.random.Generator]] = None,
 ):
     """This function creates a QCircuit with a specified number of qubits and gates.
     The gates are chosen randomly from the provided list of native gate classes.
 
     args:
         nb_qubits : Number of qubits in the circuit.
         gate_classes : List of native gate classes to use in the circuit.
-        nb_gates : Number of gates to add to the circuit. Defaults to a random
-         integer between 5 and 10.
+        nb_gates : Number of gates to add to the circuit.
+        seed: Seed used to control the random generation of the circuit.
```

Comment:
Maybe precise the behavior when the optional seed is not set (random uniform int ?)